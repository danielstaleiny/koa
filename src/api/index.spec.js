const app = require('../server')
const request = require('supertest')
const mongoose = require('../../lib/db/mongoose')

afterAll(async () => {
    mongoose.mongoose.connection.close()
})

test('ping should return pong', async () => {
    const res = await request(app.callback()).get('/ping')
    expect(res.body).toMatchSnapshot()
    expect(res.body.msg).toBe('pong')
})
