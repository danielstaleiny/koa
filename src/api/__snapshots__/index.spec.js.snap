// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ping should return pong 1`] = `
Object {
  "msg": "pong",
}
`;
